commit c529964024850038f142f0a93c6c343bb910641f
Author: Michal Nowak <mnowak@isc.org>
Date:   Tue Dec 7 17:35:01 2021 +0100

    Update brp-python-hardlink path
    
    Since Fedora 35 "brp-python-hardlink" script is in /usr/lib/rpm/redhat/.
    Otherwise RPM build fails with:
    
        + /usr/lib/rpm/brp-python-hardlink
        /var/tmp/rpm-tmp.VsVGRP: line 312: /usr/lib/rpm/brp-python-hardlink: No such file or directory
    
diff --git a/rpm/macros.scl b/rpm/macros.scl
index 550243e..c007cfd 100644
--- a/rpm/macros.scl
+++ b/rpm/macros.scl
@@ -91,7 +91,7 @@ package or when debugging this package.
     }
     /usr/lib/rpm/brp-strip-static-archive %{__strip}
     /usr/lib/rpm/brp-scl-python-bytecompile %{__python} %{?_python_bytecompile_errors_terminate_build} %{_scl_root}
-    /usr/lib/rpm/brp-python-hardlink
+    [ -f /usr/lib/rpm/redhat/brp-python-hardlink ] && /usr/lib/rpm/redhat/brp-python-hardlink || /usr/lib/rpm/brp-python-hardlink
 %{nil}}
 BuildRequires: scl-utils-build
 %if "%{?scl}%{!?scl:0}" == "%{pkg_name}"
